synapse_data_warehouse/database_roles/V2.31.0__database_access_roles.sql

```diff
@@ -0,0 +1,22 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
+
+-- Create database roles which will own the respective namespace's objects
+CREATE OR REPLACE DATABASE ROLE SYNAPSE_ALL_ADMIN;
+CREATE OR REPLACE DATABASE ROLE SYNAPSE_RAW_ALL_ADMIN;
+CREATE OR REPLACE DATABASE ROLE SCHEMACHANGE_ALL_ADMIN;
```

Comment:
I use `CREATE OR REPLACE` instead of `CREATE IF NOT EXISTS` because if we were to rollback/replay these changes, we would want our database roles to be privilege-free.

Reply:
Just to clarify: `synapse_data_warehouse_admin` (and its `dev` counterpart) is an account role, so this will be created manually, right? All the admin account roles are created manually?